hazelcast/src/main/java/com/hazelcast/internal/util/StringUtil.java

```diff
@@ -114,14 +114,58 @@ public static boolean isNullOrEmpty(String s) {
      * @param s the string to check.
      * @return true if the string is {@code null} or empty, false otherwise
      */
-
     public static boolean isNullOrEmptyAfterTrim(String s) {
         if (s == null) {
             return true;
         }
         return s.trim().isEmpty();
     }
 
+    /**
+     * Check if all Strings are not blank
+     * @param values the strings to check
+     * @return true if all the strings are not {@code null} and not blank, false otherwise
+     */
+    public static boolean isAllNotBlank(String... values) {
```

Comment:
Maybe we should call it similar to other methods, like `isAllNullOrEmptyAfterTrim()`?